### ISSUE TITLE
Pasting text inside tree or list should search 

### ISSUE THREAD

**Issue by brunosaboia** [microsoft/vscode#59455]
## TL;DR
If I have a text in my clipboard, I would like to be able to past it when I open the breadcrumb context and search for that path.

## Description
VS Code already allows breadcrumbs search. For that, you need to actually type the text you want. The search bar accepts pasting input from clipboard, but if you want to paste exactly what you have in your clipboard, you first need to type something, so the search context appears, then you delete your input and paste your desired code.

## Current behavior
Pasting over breadcrumbs context does nothing.

## Expected behavior
Pasting text over breadcrumbs context search for the clipboard content.
